app/main.py

```diff
@@ -1,8 +1,37 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort: int, mark: int, brand: str) -> None:
+        self.comfort_class = comfort
+        self.clean_mark = mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, dist: float, pwr: int, rate: float, count: int) -> None:
+        self.distance_from_city_center = dist
+        self.clean_power = pwr
+        self.average_rating = rate
+        self.count_of_ratings = count
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            if self.clean_power > car.clean_mark:
+                income += round(self.calculate_washing_price(car), 1)
```

Comment:
You don't need to use round in this case because your function already return rounded value